contrib/create-basic-configdrive

```diff
@@ -26,17 +26,15 @@ This tool creates a basic config-drive ISO image.
 CLOUD_CONFIG="#cloud-config
 
 coreos:
-  etcd2:
+  etcd-member:
```

Comment:
Does this change work?  It is not listed: https://github.com/coreos/coreos-cloudinit/blob/master/config/config.go#L39